src/operators/aggregate.h

```diff
@@ -132,6 +132,30 @@ class Aggregate : public BaseAggregate {
    */
   void execute(Task* ctx) override;
 
+  void Clear() override;
+
+  void set_prev_result(std::shared_ptr<OperatorResult> prev_result) {
```

Comment:
You can make all of these small functions inline. A good rule is less than 10 lines should be inlined (so mutators are prime candidates)